test/suites/load-balancer.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Documentation   Load balancer test suite.
+
+Library         Process
+
+Resource        ../resources/common.resource
+Resource        ../resources/kubeconfig.resource
+
+Suite Setup     Setup Suite With Namespace
+Suite Teardown  Teardown Suite With Namespace
+
+*** Variables ***
+${USHIFT_HOST}    ${EMPTY}
```

Comment:
Lets move commonly used variables to a variablefile, so multiple tests can use them and we don't have variable definitions spread throughout the .robot test suite files. See my comment on the Makefile for an example.